core/src/style/pointStyle.cpp

```diff
@@ -115,10 +117,16 @@ PointStyle::Parameters PointStyle::applyRule(const DrawRule& _rule, const Proper
 void PointStyle::pushQuad(std::vector<Label::Vertex>& _vertices, const glm::vec2& _size, const glm::vec2& _uvBL,
                           const glm::vec2& _uvTR, unsigned int _color, float _extrudeScale) const {
 
-    _vertices.push_back({{    0.0,       0.0}, {_uvBL.x, _uvTR.y}, {-1.f,  1.f, _extrudeScale}, _color});
-    _vertices.push_back({{_size.x,       0.0}, {_uvTR.x, _uvTR.y}, { 1.f,  1.f, _extrudeScale}, _color});
-    _vertices.push_back({{    0.0,  -_size.y}, {_uvBL.x, _uvBL.y}, {-1.f, -1.f, _extrudeScale}, _color});
-    _vertices.push_back({{_size.x,  -_size.y}, {_uvTR.x, _uvBL.y}, { 1.f, -1.f, _extrudeScale}, _color});
+    float es = _extrudeScale;
+
+    // Attribute will be normalized - scale to max short;
+    glm::vec2 uvTR = _uvTR * texture_scale;
+    glm::vec2 uvBL = _uvBL * texture_scale;
```

Comment:
Right, uv attributes for point, line and polygon styles could be treated the same at the moment. The text style is different. There we need to do the mapping from pixel to 0..1 in the shader since the texture size changes when the glyph atlas grows.

Do you prefer a custom scale factor as it's done for lines and polys (which provides enough precision for the largest textures and would allow proper wrapping) or use the normalize-on-the-go option of point style?